tools/install.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# MIT License
+#
+# Copyright (c) 2018 David Pinheiro
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+set -o pipefail
+
+repository="davrodpin/mole"
+install_path="/usr/local/bin"
+curl_timeout_seconds=60
+
+# Get the os architecture
+os_arch=$(uname -m)
+
+# Check if architecture is supported
+if [ "${os_arch,,}" != "x86_64" ]; then
```

Comment:
`macOs` is still complaining about `invalid substitution`. I am wonder if we really need any string substitution since the output os `uname -m` is enough:

```suggestion
if [ "$os_arch" != "x86_64" ]; then
```